core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.ts

```diff
@@ -85,26 +87,53 @@ angular.module('oppia').directive('storyViewerChaptersList', [
 
           $anchorScroll.yOffset = -80;
 
-          ctrl.setIconHighlight = function(index) {
-            ctrl.activeHighlightedIconIndex = index;
+          ctrl.isVisibleCard = function(storyNode) {
+            if (!storyNode) {
+              return false;
+            }
+            var isNextPendingNode = (
```

Comment:
This is a boolean, so try to rename this so that it does not start with a verb.

Reply:
Removed this variable.